build-tests/heft-sass-test/config/typescript.json

```diff
@@ -52,7 +52,7 @@
     /**
      * File extensions that should be copied from the src folder to the destination folder(s).
      */
-    "fileExtensions": [".sass", ".scss", ".css"]
+    "fileExtensions": []
```

Comment:
It should probably still copy `.css` files.

Reply:
CSS files go through the same pipeline as the scss files, so copying is taken care of by the sass plugin.